opre_ops/opre_ops/settings/cloudgov.py

```diff
@@ -0,0 +1,27 @@
+"""
+Configuration for running OPRE OPS in cloud.gov.
+"""
+import os
+
+# Import all common settings relevant to both local & cloud:
+from opre_ops.settings.common import *
+
+# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.environ.get('DATABASE_NAME'),
+        'USER': os.environ.get('DATABASE_USER'),
+        'PASSWORD': os.environ.get('DATABASE_PASSWORD'),
+        'HOST': os.environ.get('DATABASE_HOST'),
+        'PORT': '5432'
+    }
```

Comment:
I remember we can't expect the user-defined environment variables to stay put after redeployment to cloud.gov (I ran into that issue about 1-2 years ago).  Instead we will need to use [user-provided service credentials](https://docs.cloudfoundry.org/devguide/services/user-provided.html).

But I don't think we need to do that, cloud.gov provides the variable called [`DATABASE-URL`](https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#DATABASE-URL) that we can use for this purpose.

I think that's what [`dj-database-url`](https://github.com/jacobian/dj-database-url/blob/master/test_dj_database_url.py#L149-L158) is for.  It takes `DATABASE_URL` and pulls out the necessary information, which is in cloud.gov environment.  This will work because it takes that variable and populates everything correctly.

After taking a look at [my other example](https://github.com/18F/usda-fns-ingest/blob/master/usda_fns_ingestor/usda_fns_ingestor/settings/base.py#L78) in previous project, now that makes sense.  `default` within `config` is only used for local db, because there's no need for hostname/username/password, only the name of the database (that's why there's three `/`s.  But in cloud.gov settings, the `default` is not used, it will take cloud.gov's `DATABASE_URL`.

I would either consider giving `dj-database-url` another try, or we can pull out information from `DATABASE_URL` from cloud.gov and do essentially the same thing to fill out this information.

Reply:
Very helpful, thank you @amymok!

Reply:
@amymok I updated this PR so that it now reads in the cloud.gov configuration from `VCAP_SERVICES`. Appreciate the pointers here, I did not realize that setting ENV variables through the cloud.gov UI/CLI wasn't the way to go! I didn't implement `dj-database-url` because my rule of thumb is to avoid third-party libraries when it's easy/straightforward to do so. Please take another look when you have time!